openedx/features/content_tagging/toggles.py

```diff
@@ -0,0 +1,17 @@
+
+"""
+Toggles for content tagging
+"""
+
+from edx_toggles.toggles import WaffleSwitch
+
+# .. toggle_name: content_tagging.auto
+# .. toggle_implementation: WaffleSwitch
+# .. toggle_default: False
+# .. toggle_description: Setting this enables automatic tagging of content
+# .. toggle_type: feature_flag
+# .. toggle_category: admin
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: 2023-08-30
+# .. toggle_tickets: https://github.com/openedx/modular-learning/issues/79
+CONTENT_TAGGING_AUTO = WaffleSwitch('content_tagging.auto', __name__)
```

Comment:
Maybe we should use `CourseWaffleFlag` so that we can turn this on on a per-course basis? What do you think?

Reply:
Will this feature be enabled by default sometime in the future? Or do you see it always as an opt-in? 
Wouldn't it be a hassle to make the user enable it every time he creates a course?

I don't know the product well enough to find the right balance between features and usability here. I'll happily follow your call!

Reply:
Yes, in the future we'll enable it by default and remove this setting. Making it a course waffle flag doesn't mean it's only enabled on a course-by-course basis; you can still turn it on for the whole instance, on our sandbox for example. It just means that during the testing phase you _may_ choose to only turn it on for certain courses.

Reply:
Great! I will look at the documentation tomorrow and change it to a `CourseWaffleFlag` if I don't see any problem.

Thank you for your input!

Reply:
Done 154a4aec8b6117a8701af74ec216f086f023bdd7!